website/pages/style.css

```diff
@@ -89,16 +87,16 @@ footer .content {
 .g-section-block section {
   padding-top: 96px;
   padding-bottom: 96px;
-}
 
-.g-section-block section > .g-section-header + *,
```

Comment:
This fixes an issue where a parent selector was redeclared while already being adjacent to itself.